backend/ohq/statistics.py

```diff
@@ -92,20 +93,22 @@ def calculate_num_students_helped(queue, prev_sunday, coming_saturday):
 
 
 def calculate_questions_per_ta_heatmap(queue, weekday, hour):
-    interval_questions = Question.objects.filter(
-        queue=queue, time_asked__week_day=weekday, time_asked__hour=hour
+    interval_stats = (
+        Question.objects.filter(queue=queue, time_asked__week_day=weekday, time_asked__hour=hour)
+        .annotate(date=TruncDate("time_asked"))
+        .values("date")
+        .annotate(
+            q_per_ta=1.0 * Count("date", distinct=False) / Count("responded_to_by", distinct=True)
```

Comment:
Discovered another potential bug on my test db - responded_to_by could be 0 if there happens to be a slot with one question asked by never responded by anyone which causes a divide-by-zero error. 

Using a case/when/default should work here. 
https://docs.djangoproject.com/en/3.1/ref/models/conditional-expressions/

Reply:
Can we also add a test case to catch this regression?